File: basic_questions/printHollowRect/printHollowRect.java
package basic_questions.printHollowRect;

public class printHollowRect {
    public static void main(String[] args) {
        int rows = 4;
        int column = 5;
        
        //rows 
        for (int i = 1 ; i<=rows ; i++){
            for(int j =1 ; j<=column;j++){
                if (i==1 || j==1 || i==rows||j==column){
                    System.out.print("*");
                }
                else{
                    System.out.print(" ");
                }
            }
            System.out.println();
        }
    }
}

File: basic_questions/printHollowRect/printHollowRect.go
package main

import "fmt"

func main() {

	rows := 4
	column := 5

	for i := 1; i <= rows; i++ {
		for j := 1; j <= column; j++ {
			if i==1|| j==1 || i==rows|| j==column{

				fmt.Print("*")
			}else{

				fmt.Print(" ")
			}
		}
		fmt.Println()
	}
}
